src/_common/fireside/post/post-model.ts

```diff
@@ -511,3 +520,12 @@ export class FiresidePost extends Model implements ContentContainerModel, Commen
 }
 
 Model.create(FiresidePost);
+
+/**
+ * Will load the article from the API and store it into the post model.
+ */
+export async function loadArticleIntoPost(post: FiresidePost) {
+	const payload = await Api.sendRequest(`/web/posts/article/${post.id}`);
+	post.article_content = payload.article;
+	return post;
+}
```

Comment:
Remember, we want to switch to more "functional" style so that things can be imported into the chunks they're used. When it's all instance functions that code gets imported always.